db/images.go

```diff
@@ -148,18 +148,20 @@ func HasImage(id uint64) (has bool, err error) {
 	return
 }
 
-// InsertImage insert and image into and existing open post and return image
+// InsertImage insert an image into an existing open post and return image
 // JSON
-func InsertImage(tx *sql.Tx, postID uint64, token, name string, spoiler bool,
+func InsertImage(
+	tx *sql.Tx, postID uint64, token, name string, spoiler, mask bool,
```

Comment:
The way I se see it, there is no need to alter this function. Why would we need to store, if the image was masked? Just have the client send the already masked name.

Reply:
>Why would we need to store, if the image was masked?

It wasn't stored. It was used for some logic in the sql query.